apps/frontend/src/main/template-editor-service.ts

```
@@ -0,0 +1,325 @@
+        } else if (event.type === 'content_block_delta') {
+          if (event.delta.type === 'text_delta') {
+            assistantMessage += event.delta.text;
+            this.emit('stream-chunk', templateId, { type: 'text', text: event.delta.text });
+          }
+        } else if (event.type === 'content_block_start' && event.content_block.type === 'tool_use') {
+          currentToolUse = {
+            id: event.content_block.id,
+            name: event.content_block.name,
+            input: {}
+          };
+        } else if (event.type === 'content_block_delta' && event.delta.type === 'input_json_delta') {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)